src/WallGo/config.py

```diff
@@ -0,0 +1,244 @@
+    def loadConfigFromFile(self, filePath: str) -> None:
+        """
+        Load the configs from a file.
+
+        Parameters
+        ----------
+        filePath : str
+            Path of the file where the configs are.
+
+        """
+        parser = ConfigParser()
+        parser.readINI(filePath)
+        
+        # Read the Grid configs
+        self.configGrid = ConfigGrid(
+            spatialGridSize=parser.getint("Grid", "spatialGridSize"),
+            momentumGridSize=parser.getint("Grid", "momentumGridSize"),
+            ratioPointsWall=parser.getfloat("Grid", "ratioPointsWall"),
+            smoothing=parser.getfloat("Grid", "smoothing"),
+        )
+
+        # Read the EOM configs
+        self.configEOM = ConfigEOM(
+            errTol=parser.getfloat("EquationOfMotion", "errTol"),
+            pressRelErrTol=parser.getfloat("EquationOfMotion", "pressRelErrTol"),
+            maxIterations=parser.getint("EquationOfMotion", "maxIterations"),
+            conserveEnergyMomentum=parser.getboolean(
+                "EquationOfMotion", "conserveEnergyMomentum"),
+            wallThicknessBounds=(
+                parser.getfloat("EquationOfMotion", "wallThicknessLowerBound"),
+                parser.getfloat("EquationOfMotion", "wallThicknessUpperBound")),
+            wallOffsetBounds=(
+                parser.getfloat("EquationOfMotion", "wallOffsetLowerBound"),
+                parser.getfloat("EquationOfMotion", "wallOffsetUpperBound")),
+            vwMaxDeton=parser.getfloat("EquationOfMotion", "vwMaxDeton"),
+            nbrPointsMinDeton=parser.getint("EquationOfMotion", "nbrPointsMinDeton"),
+            nbrPointsMaxDeton=parser.getint("EquationOfMotion", "nbrPointsMaxDeton"),
+            overshootProbDeton=parser.getfloat("EquationOfMotion","overshootProbDeton"),
+        )
+
+        # Read the Hydrodynamics configs
+        self.configHydrodynamics = ConfigHydrodynamics(
+            tmin=parser.getfloat("Hydrodynamics", "tmin"),
+            tmax=parser.getfloat("Hydrodynamics", "tmax"),
+            relativeTol=parser.getfloat("Hydrodynamics", "relativeTol"),
+            absoluteTol=parser.getfloat("Hydrodynamics", "absoluteTol"),
+        )
+
+        # Read the Thermodynamics configs
+        self.configThermodynamics = ConfigThermodynamics(
+            tmin=parser.getfloat("Thermodynamics", "tmin"),
+            tmax=parser.getfloat("Thermodynamics", "tmax"),
+            phaseTracerTol=parser.getfloat("Thermodynamics", "phaseTracerTol"),
+        )
+
+        # Read the BoltzmannSolver configs
+        self.configBoltzmannSolver = ConfigBoltzmannSolver(
+            collisionMultiplier=parser.getfloat(
+                "BoltzmannSolver", "collisionMultiplier"),
+            basisM=parser.get("BoltzmannSolver", "basisM"),
+            basisN=parser.get("BoltzmannSolver", "basisN"),
+        )
+
+class ConfigParser:
+    """class Config -- Manages configuration variables for WallGo. This is essentially a

```

Comment:
I think you can remove this, currently it doesn't really do anything that configparser.ConfigParser doesn't already